src/docfx/template/MustacheXrefTagParser.cs

```diff
@@ -28,37 +27,35 @@ public static string ProcessXrefTag(string templateStr)
 
         private static string ReplaceXrefTag(Match match)
         {
-            var xrefTag = HtmlUtility.LoadHtml(match.Value).ChildNodes.FindFirst("xref");
             var uidName = "uid";
             var partialName = default(string);
-            foreach (var attribute in xrefTag.Attributes)
+            var reader = new HtmlReader(match.Value);
+            while (reader.Read(out var token))
             {
-                // TODO: uid may fallback to href in ProfileList
-                if (string.Equals(attribute.Name, "uid", StringComparison.OrdinalIgnoreCase))
+                if (token.NameIs("xref"))
```

Comment:
would there be multiple `xref` nodes? This would take the last, and the previous implementation took the first.

Reply:
or return when found `xref`?

Reply:
not likely, because the regex uses a non greedy match, it'll grab one `<xref>` tag at a time.